GoogleCloudPlatform/bank-of-anthos
# cleanup: change css to minimize the alert box

### Fixes #419

### Background 
- Update the css to minimize the alert box for deposit

### Change Summary
- Override inherited padding to make the box smaller

### Additional Notes
- N/A

### Testing Procedure
- View changes in my cluster: http://35.247.96.212/
- You can either host your own cluster or check in the CI cluster
- Create port forward to the frontend: `kubectl port-forward <POD_ID> -n <NAMESPACE> 8080:8080`
- Access the Pod via `localhost:8080`
- Make a deposit and see if the alert box is compact compared to the one in http://bank-of-anthos.xyz/


### Related PRs or Issues 
- N/A

### Before
![image](https://user-images.githubusercontent.com/7249208/145909174-58e8127b-1fd3-4cc4-9ad4-cae383acce56.png)

### After
![image](https://user-images.githubusercontent.com/7249208/145908962-9c355712-8603-4412-9e40-75e097289a5e.png)
